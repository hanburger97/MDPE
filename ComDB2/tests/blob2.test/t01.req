insert into t1 values(4, NULL, x'1234abcd')
insert into t1 values(5, x'1234abcd', x'1234abcd')
select length(data), length(inline_data) from t1 order by id
