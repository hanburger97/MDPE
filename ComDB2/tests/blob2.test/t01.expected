(rows inserted=1)
(rows inserted=1)
(length(data)=6, length(inline_data)=6)
(length(data)=6, length(inline_data)=6)
(length(data)=6, length(inline_data)=6)
(length(data)=NULL, length(inline_data)=4)
(length(data)=4, length(inline_data)=4)
